src/Directory.Build.props

```diff
@@ -6,6 +6,7 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <DisableImplicitFrameworkDefines>true</DisableImplicitFrameworkDefines>
     <DisableImplicitConfigurationDefines>true</DisableImplicitConfigurationDefines>
+    <GenerateResourceUsePreserializedResources>false</GenerateResourceUsePreserializedResources>
```

Comment:
@ericstj I had to this to get rid of "Preserialized resources require the System.Resources.Extensions assembly at runtime, but it was not found in this project's references." error emitted for every project otherwise.

Is this the right way to suppress this error?

Is it a bug that the error is emitted even for plain vanilla text resources? I do not think we want everybody with plain vanilla text resources to add dependency on System.Resources.Extensions package.

Reply:
Yes, that's what you'll need to do.  Interesting side effect.  @rainersigwald / @nguerrera this is happening because the condition for `GenerateResourceUsePreserializedResources`  is based solely on `netcoreapp3.0` TargetFramework.  I wonder if we should change the behavior here to support the cases where folks set `DisableImplicitFrameworkReference`.  What if we used `GenerateResourceUsePreserializedResources` as a soft condition, then we'd still let folks have the old behavior if Resources.Extensions wasn't referenced?  If we did encounter a resource that required it we could tell them what was missing (either setting the property, or referencing RE or both).

Reply:
Yes, I think the error should only trigger if there is a non-string encountered in addition to no System.Resources.Extensions. I thought we discussed that, but maybe didn't quite get it right in the implementation. File an issue on msbuild?

Reply:
The old behavior in the face of non-strings was bogus, it would emit the serialized data as a string. So, if you have a non-string moving forward in core msbuild, it should require System.Resources.Extensions and GenerateResourceUsePreserializedResources

Reply:
But if you only have strings, it should just work as before.

Reply:
Sure.  Filed https://github.com/microsoft/msbuild/issues/4474